frontend/src/metabase/dashboard/components/DashCard/DashCardActionButtons/index.ts

```diff
@@ -1,2 +1 @@
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export { default } from "./DashCardActionButtons";
+export { DashCardActionsPanel } from "./DashCardActionsPanel";
```

Comment:
> - 06d277e312ca023f99506aac1fc07b340779e461 moves `DashboardCardActionsPanel` inside the `DashCardActionButtons` component as it's basically just a styled div.
> - e0d9ef004c5b4ef24ed829d6f755833f143aa3ff renames `DashCardActionButtons` to `DashCardActionsPanel` as now that component contains the whole panel and I think it's a more appropriate name for it

Now the directory is called `DashCardActionButtons` but the the only component it's exporting is `DashCardActionsPanel` (which does not mention `Buttons` in its name).

I think - despite props passing - we were better off with `DashboardCardActionsPanel` definition living in `DashCard`.

Reply:
Should I close the PR or is it just a matter of finding a better name?

Reply:
> Should I close the PR 

No, the PR gets rid of `renderDashCardActions` which is a good thing.

> is it just a matter of finding a better name?

A better name is one way to handle it.
Another is to keep the `isDashCardTabMenuOpen` state and `DashboardCardActionsPanel` definition in `DashCard`.
Also we could keep it as is, it's definitely not a blocking comment.

Reply:
I restructured the files a bit, this is the resulting DashCard folder
<img width="321" alt="image" src="https://github.com/metabase/metabase/assets/1914270/dbdc4036-956f-4203-8e76-c6a450c85e9e">
The rationale is that `DashCardActionsPanel` uses all the buttons, that are now children of the `DashCardActionsPanel` folder (which is pretty much the same as renaming the `DashCardActionButtons` folder).
I also created some a folder for some of the buttons that were in a single file, in this way it should be clear that DashCardActionsPanel.tsx is the main component.

Do you think this looks better?

Reply:
Oh, nice! :+1: